cpp/inc/bond/core/transforms.h

```diff
@@ -388,6 +388,12 @@ class RequiredFieldValiadator
         _required = next_required_field<typename schema<T>::type::fields>::value;
     }
 
+    void Base(bool done) const
+    {
+        if (done)
```

Comment:
Minor: slight preference for curly braces always.

Reply:
Was trying to be consistent here.